mirgecom/artificial_viscosity.py

```diff
@@ -0,0 +1,328 @@
+r""":mod:`mirgecom.artificial_viscosity` Artificial viscosity for hyperbolic systems.
+
+Consider the following system of conservation laws of the form:
+
+.. math::
+
+    \partial_t \mathbf{Q} + \nabla\cdot\mathbf{F} = 0,
+
+where $\mathbf{Q}$ is the state vector and $\mathbf{F}$ is the vector of
+fluxes. This module applies an artificial viscosity term by augmenting
+the governing equations in the following way:
+
+.. math::
+
+    \partial_t \mathbf{Q} + \nabla\cdot\mathbf{F} =
+    \nabla\cdot{\varepsilon\nabla\mathbf{Q}},
+
+where $\varepsilon$ is the artificial viscosity coefficient.
+To evalutate the second order derivative numerically, the problem
+is recast as a set of first order problems:
+
+.. math::
+
+    \partial_t{\mathbf{Q}} + \nabla\cdot\mathbf{F} &= \nabla\cdot\mathbf{R} \\
+    \mathbf{R} &= \varepsilon\nabla\mathbf{Q}
+
+where $\mathbf{R}$ is an auxiliary variable, and the artitifial viscosity
```

Comment:
Missed this, adding it for a a batch fix
```suggestion
where $\mathbf{R}$ is an auxiliary variable, and the artificial viscosity
```